### PR TITLE
Advance mode for AirebaseNG added

### PR DESCRIPTION
**Pull request by AnkitB125**
Added the advance mode option for the airebaseNG by adding wep key argument to encrypt or decrypt the packets

### CODE REVIEW

**Comment by zkkok** [comment]
Hi @AnkitB125 ,

Thanks for your contribution. 

Please note that we use the trello board as the main way of assigning tasks. In the future, please go through the trello board to assign yourself the task you want to be working on and communicate with your peers currently working on it before commencing. This is to avoid collision.

Also, please generate the branch from the original issue.

File: src/components/AirbaseNG/AirbaseNG.tsx
Comment (bradleybogg) [inline src/components/AirbaseNG/AirbaseNG.tsx:1]: Changes look good. Pending @gangamarx application testing.

Comment (gangamarx) [reply src/components/AirbaseNG/AirbaseNG.tsx:1]: @AnkitB125 airbase-advance-mode
I am seemed to get an error when I did the testing both in standard mode and advanced mode in my laptop. Upon researching the error message, it says the SSID or WLAN meant to be in quotes as there is a space in WLAN etc. Please advice if I need to setup anything correctly. Attached is the error msg screenshot. 
[Standard test error.docx](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/files/15021358/Standard.test.error.docx)

**Review by gangamarx — COMMENTED** [review]
@AnkitB125 airbase-advance-mode
I am seemed to get an error when I did the testing both in standard mode and advanced mode in my laptop. Upon researching the error message, it says the SSID or WLAN meant to be in quotes as there is a space in WLAN etc. Please advice if I need to setup anything correctly. Attached is the error msg screenshot. 
[Standard test error.docx](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/files/15021358/Standard.test.error.docx)

**Review by bradleybogg — CHANGES_REQUESTED** [review]
Please rectify the above requested changes.

**Comment by AnkitB125** [comment]
I did not tested the airbaseNG on my system since I am using Window and I followed the following guide to implement the advance mode
[MN503_Assignment_2_T1_2024.pdf](https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/files/15220472/MN503_Assignment_2_T1_2024.pdf)

**Comment by michaeljpigott** [comment]
Hi @AnkitB125, can you please resolve the conflicts with this branch? Click on _Resolve conflicts_ for more information. You also need to figure out the issue raised by Ganga.

**Comment by michaeljpigott** [comment]
This pull request is out-of-date (from Trimester 1, 2024).

**Pull request closed without merging**